docusaurus/docs/explore/rpc.md

```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
 poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.poktroll.com
 ```
 
-### gRPC
+## Alpha TestNet
+
+### Alpha RPC Endpoints
+
+We have provided `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
 
-_TODO_TECHDEBT: Add a gRPC example_
+- **RPC**: `https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com`
+- **gRPC**: `https://shannon-testnet-grove-seed-grpc.alpha.poktroll.com`
+- **REST**: `https://shannon-testnet-grove-seed-api.alpha.poktroll.com`
 
-### REST
+### Alpha JSON-RPC Example
 
-_TODO_TECHDEBT: Add a REST example_
+Using `curl`:
+
+```bash
+curl -X POST https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com/block
+```
+
+Using the `poktrolld` binary:
+
+```bash
+poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com
```

Comment:
```suggestion
poktrolld query block --type=height 1 --node https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com
```
Using height `0` returns an error. Could you please update it for all occurrences in this doc?